src/containers/login/login.component.jsx

```diff
@@ -1,19 +1,7 @@
-import React, { useEffect, useState } from "react";
-import { auth } from "database";
-import { useSelector } from "react-redux";
-import Home from "containers/home";
+import React from "react";
 
 const Login = (props) => {
-  const [user, setUser] = useState({});
-
-  useEffect(() => {
-    setUser(userFromState);
-  }, []);
-
-  const userFromState = useSelector((state) => state.users.user);
-  console.log("state", user);
-  console.log(userFromState.full_name);
-  return !userFromState.full_name ? (
+  return (
```

Comment:
Sweet cleanup 🤙

Reply:
Dont get this one

Reply:
Lol I mean good job cleaning that part of the code up